pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -143,6 +143,18 @@ type Database struct {
 	// Defines the resource requests and limits for the database pods.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+	// Defines the interval for backups in cron expression format.
+	// +optional
+	BackupSchedule string `json:"backupSchedule,omitempty"`
+	// Defines the object storage url for uploading backups.
+	// +optional
+	BackupURL string `json:"backupURL,omitempty"`
+	// Defines the backup retention policy.
+	// +optional
+	BackupRemoteDeletePolicy string `json:"backupRemoteDeletePolicy,omitempty"`
+	// Defines the secret to be used for uploading backup.
+	// +optional
+	BackupSecret string `json:"backupSecret,omitempty"`
```

Comment:
A small request: could we rename this to match the syntax from the mysql operator `BackupSecretName`?

It's the only one that is slightly different so might as well make them all align.

Reply:
Yes makes sense. I will make the change :)